readme.md

```diff
@@ -0,0 +1,123 @@
+# Authsignal Server Python SDK
+
+[Authsignal](https://www.authsignal.com/?utm_source=github&utm_medium=python_sdk) provides passwordless step up authentication (Multi-factor Authentication - MFA) that can be placed anywhere within your application. Authsignal also provides a no-code fraud risk rules engine to manage when step up challenges are triggered.
+
+## Installation
+
+Python 3
+
+```bash
+pip3 install authsignal
+```
+
+or install newest source directly from GitHub:
+
+```bash
+pip3 install git+https://github.com/authsignal/authsignal-python
+```
+
+## Configuration
+Initialize the Authsignal Python SDK, ensuring you do not hard code the Authsignal Secret Key, always keep this safe.
+
+```python
+import authsignal.client
+
+client = authsignal.Client(api_key='<SECRET API KEY HERE>')
```

Comment:
Everyone calls there things `client` or `app` - for easier copy/pasting I'd be explicit even in this readme.
```suggestion
authsignal_client = authsignal.Client(api_key='<SECRET API KEY HERE>')
```